JDI.Light/JDI.Light/Factories/WebDriverFactory.cs

```diff
@@ -153,19 +153,13 @@ public string RegisterDriver(string driverName)
             }
         }
 
+
         public void SetRunType(string runType)
         {
-            switch (runType)
-            {
-                case "remote":
-                    RunType = RunType.Remote;
-                    return;
-                default:
-                    RunType = RunType.Local;
-                    return;
-            }
-
-            RunType = RunType.Local;
+            if (runType == "remote")
+                RunType = RunType.Local;
+            else
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5844553)